src/sbt-test/docker/override-commands/build.sbt

```diff
@@ -9,7 +9,7 @@ version := "0.1.0"
 maintainer := "Gary Coady <gary@lyranthe.org>"
 
 dockerCommands := Seq(
-  Cmd("FROM", "dockerfile/java:latest"),
+  Cmd("FROM", "dockerfile/openjdk:latest"),
```

Comment:
Not sure I'm following, this is just a cherry-pick from master. There shouldn't be anything to apply on master, no?